### DISCUSSION TITLE
Location Option Acting Weird

### DISCUSSION THREAD

**Discussion by Unsantae** [firefly-iii/firefly-iii#7681] [Firefly III] [answered]
Recently, the MapBox integration has stopped working for me.  Anyone else?  The API key is correctly entered in my .env file and the Location option is enabled in preferences.
![Screen Shot 2023-06-24 at 12 06 39 PM](https://github.com/firefly-iii/firefly-iii/assets/122224780/633c442f-cb31-48ea-8898-f92efd994737)
![Screen Shot 2023-06-24 at 12 07 10 PM](https://github.com/firefly-iii/firefly-iii/assets/122224780/00aca8c9-1e8a-4fc2-a0ae-d095c3ed4b3f)

For items that I know already had a location stored, the Location box shows up but is blank.
![Screen Shot 2023-06-24 at 12 02 42 PM](https://github.com/firefly-iii/firefly-iii/assets/122224780/3c1528dd-3187-4e6d-ac4e-08b0909106b6)
For new items, there is no option to add a location at all.
![Screen Shot 2023-06-24 at 12 05 25 PM](https://github.com/firefly-iii/firefly-iii/assets/122224780/71a3f0b5-9c4a-477b-96bd-c6b572f1a22c)

For context, I only use the mapping feature for tags.  Not transaction, accounts, or anything else.

**Comment by JC5** [comment, marked as answer]
No that's on me. The library was acting weird and the implementation is not really documented. I dropped most of the code and hope to restore it soon ™️

**Reply by Unsantae** [reply]
Thank you sir.  I had been messing around in the database backend trying to clean up a bunch of deleted accounts/transactions/tags from initial imports.  Even though I’ve been very careful with the SQL queries, I thought I might have messed something up.  Good to know I hadn’t!

**Reply by edmundo096** [reply]
This is unfortunate but understandable.
As a new user, I got some hard time looking for what was wrong, such as why the Location checkbox was not doing anything and looking my configuration.

A note on the checkbox (as well as maybe also disabling it) and on the docs, about having it temporally disabled, would be useful.
